test/setup.ts

```diff
@@ -0,0 +1,3 @@
+import fetch from 'node-fetch';
```

Comment:
This throw some TS error for me
```sh
 FAIL  test/index.test.ts
  ● Test suite failed to run

    test/setup.ts:3:1 - error TS2322: Type '(url: RequestInfo, init?: RequestInit | undefined) => Promise<Response>' is not assignable to type '(input: RequestInfo | URL, init?: RequestInit | undefined) => Promise<Response>'.
      Types of parameters 'url' and 'input' are incompatible.
        Type 'RequestInfo | URL' is not assignable to type 'RequestInfo'.
          Type 'Request' is not assignable to type 'RequestInfo'.
            Type 'Request' is missing the following properties from type 'Request': size, buffer

    3 global.fetch = fetch;
      ~~~~~~~~~~~~
```
This did the trick
```suggestion
import fetch from 'cross-fetch';
```